lib/Sema/TypeCheckEffects.cpp

```diff
@@ -1347,21 +1352,21 @@ class ApplyClassifier {
       switch (auto polyKind = fnRef.getPolymorphicEffectKind(kind)) {
       case PolymorphicEffectKind::AsyncSequenceRethrows:
       case PolymorphicEffectKind::ByConformance: {
-        auto substitutions = fnRef.getSubstitutions();
-        auto requirements =
-            substitutions.getGenericSignature().getRequirements();
-        auto conformances = substitutions.getConformances();
+        auto requirements = substitutions.getGenericSignature()
```

Comment:
Maybe the correct check here is to skip any protocols that are not rethrows or something? I don’t think we ever got this fully working here…

Reply:
Like if you change your test case to add a FakeCopyable protocol everywhere does it stop working on 5.10?

Reply:
I think that's what classifyConformance does it seems

Reply:
For posterity, we talked about this offline and decided to land this because it's pretty narrow and shouldn't affect anything else.